addons/hunterkiller/stringtable.xml

```diff
@@ -48,6 +51,7 @@
             <German>Überschreibe</German>
             <Korean>오버라이드</Korean>
             <Japanese>オーバーライド</Japanese>
+            <Chinesesimp>超控</Chinesesimp>
```

Comment:
these seem to be the referenced lines, feel free to make a replacement suggestion

Reply:
```suggestion
            <Chinesesimp>超越控制</Chinesesimp>
```